src/dependency_manager/manager.py

```diff
@@ -1,18 +1,19 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
-import pkg_resources
 from typing import Union
+import pkg_resources
 from .singleton import Singleton
 
 
 class DependencyManagerAbstract(Singleton, ABC):
-    def init(self):
+    def init(self, dry_run=False):
```

Comment:
> Added a dry-run flag which defaults to true.

which is your intent? I think False is the right call. We want the DM to run correctly by default and change dependencies.

Reply:
False, should be the default behavior.